deploy-all-in-one-tls.yaml

```diff
@@ -236,6 +236,23 @@ data:
       # Set false to disable upgrade notification
       upgradeNotifier: true
 ---
+# pod security policy
+apiVersion: extensions/v1beta1
```

Comment:
We should be using `apiVersion: policy/v1beta1` instead of `extensions` APIGroups